<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/new/prefix1/iconos/usuario.png</normaloff>:/new/prefix1/iconos/usuario.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="area">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Inicio de sesion</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Correo</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sesioncorreo">
                <property name="maxLength">
                 <number>50</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Contraseña</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="sesioncon">
                <property name="maxLength">
                 <number>20</number>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sesionvercon">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="iconos.qrc">
                  <normaloff>:/new/prefix1/iconos/ojo-abierto.png</normaloff>:/new/prefix1/iconos/ojo-abierto.png</iconset>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QPushButton" name="pushButton_4">
                <property name="text">
                 <string>Registrarse</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sesionEntrar">
                <property name="text">
                 <string>Entrar</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Registro</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Nombre</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="nombre">
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Nombre de Usuario</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="nombreUsuario">
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Correo</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="correo">
                <property name="maxLength">
                 <number>50</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Contraseña</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="con">
                <property name="maxLength">
                 <number>20</number>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="verCon">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="iconos.qrc">
                  <normaloff>:/new/prefix1/iconos/ojo-abierto.png</normaloff>:/new/prefix1/iconos/ojo-abierto.png</iconset>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Repite tu Contraseña</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="con_2">
                <property name="maxLength">
                 <number>20</number>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="verCon_2">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="iconos.qrc">
                  <normaloff>:/new/prefix1/iconos/ojo-abierto.png</normaloff>:/new/prefix1/iconos/ojo-abierto.png</iconset>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Genero</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="gender">
                <item>
                 <property name="text">
                  <string>NULL</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>F</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>M</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Tipo de Usuario</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="tipoUsuario">
                <item>
                 <property name="text">
                  <string>USUARIO</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>INVITADO</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>EMPLEADO</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>ADMINISTRADOR</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Registrarse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Contraseña de Administrador</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="conAdministrador">
           <property name="maxLength">
            <number>20</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="regresar">
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
</ui>
